client/package.json

```diff
@@ -18,7 +18,7 @@
     "@ibm/plex": "^6.0.0",
     "@sentry/browser": "^6.3.6",
     "bpmn-js": "^9.0.3",
-    "bpmn-js-properties-panel": "^1.0.0-alpha.12",
+    "bpmn-js-properties-panel": "^1.2.0",
```

Comment:
We need to bump and release the properties panel in `camunda-bpmn-js` yet and bump it here.